backend/src/main/java/de/neuefische/backend/dto/DoctorDto.java

```diff
@@ -20,6 +20,7 @@
     private String phoneNumber;
     private String mobileNumber;
     private String emailAddress;
+    private AppointmentDto appointmentDto;
```

Comment:
I think I'd stick to the variable name appointment

Reply:
I prefer to name it "...Dto" when it is a Dto